localization/yabloc/yabloc_particle_filter/launch/yabloc_particle_filter.launch.xml

```diff
@@ -4,7 +4,7 @@
   <arg name="twist_cov_for_prediction" default="/localization/twist_estimator/twist_with_covariance"/>
   <let name="output_weighted_particles" value="weighted_particles"/>
   <let name="output_particles_marker_array" value="predicted_particles_marker"/>
-  <node pkg="yabloc_particle_filter" exec="predictor_node" name="particle_predictor" output="screen" args="--ros-args --log-level info">
```

Comment:
Just out of curiosity, why did you delete `args=”--ros-args --log-level info“`?

Reply:
In the past, I used to rewrite the info as warn or error for debugging. I erased it because of bad manners. Sorry.

Reply:
Oh, is this bad manners? I thought it was convenient and should be encouraged.
Does it have a negative effect?

Reply:
There is no particularly bad impact. 
But if the behavior is the same whether it is present or not, it would be more natural to have it consistent with the other packages.

To be honest, I don't care either way